scripts/nb_duet_test.py

```diff
@@ -57,6 +57,7 @@
     list(Path("examples/homomorphic-encryption").rglob("*.ipynb"))
     + list(Path("examples/duet/dcgan").rglob("*.ipynb"))
     + list(Path("examples/duet/super_resolution").rglob("*.ipynb"))
+    # + list(Path("examples/duet/reinforcement_learning").rglob("*.ipynb"))
```

Comment:
Can we add to notebook to the CI?

Reply:
It doesn't work, thats why its commented out. It uses asyncio in the notebook so I suspect that is colliding with the asyncio sleeping in the integration testing system.